typescript/packages/plugins/balmy/src/balmy.service.ts

```diff
@@ -0,0 +1,186 @@
+import { QuoteRequest, buildSDK } from "@balmy/sdk";
+import { Tool } from "@goat-sdk/core";
+import { EVMWalletClient } from "@goat-sdk/wallet-evm";
+import { ExecuteSwapParameters, GetAllowanceParameters, GetBalanceParameters, GetQuoteParameters } from "./parameters";
+
+type BalmyConfig = {
+    quotes: {
+        sourceList: {
+            type: "local";
+        };
+        defaultConfig: {
+            global: {
+                slippagePercentage: number;
+                txValidFor: string;
+                disableValidation: boolean;
+            };
+            custom: {
+                bebop: { enabled: boolean; apiKey: string };
+                "0x": { enabled: boolean; apiKey: string; baseUrl: string };
+                "li-fi": { apiKey: string };
+                paraswap: { enabled: boolean; sourceAllowlist: string[]; sourceDenylist: string[] };
+                "1inch": { enabled: boolean; customUrl: string; sourceAllowlist: string[] };
+            };
+        };
+    };
+    provider: {
+        source: {
+            type: "http";
+            url: string;
+            supportedChains: number[];
+        };
+        defaultConfig: {
+            chainId: number;
+            rpcUrl: string;
+        };
+    };
+};
+
+type TokenBalance = {
+    [address: string]: string;
+};
+
+type TokenBalances = {
+    [token: string]: TokenBalance;
+};
+
+type ChainBalances = {
+    [chainId: string]: TokenBalances;
+};
+
+export class BalmyService {
+    private sdk;
+
+    constructor(config: BalmyConfig) {
+        this.sdk = buildSDK(config);
+    }
+
+    @Tool({
+        description: "Get token balances for an account",
+    })
+    async getBalances(walletClient: EVMWalletClient, parameters: GetBalanceParameters) {
```

Comment:
This method is a bit useless. It only allows you to query the balance for one token which you already get in the ERC20 plugin. Why would you need this?